modules/system.nix

```diff
@@ -158,6 +158,22 @@ in
 
         mkdir -p /run
         ln -s $_system_config /run/current-system
+
+        mkdir -p /run/current-system/sw/bin
+        ${concatStringsSep "\n" (map (pkg:
+          ''
+            execs=$(${pkgs.findutils}/bin/find ${pkg}/bin -type f)
+            for exec in $execs; do
+              cat << EOF > /run/current-system/sw/bin/$(basename $exec)
+#!${pkgs.busybox}/bin/sh
+
+export PATH="$PATH"':${makeBinPath config.environment.systemPackages}'
```

Comment:
The fact that `PATH` does not contain `/bin` is a bug and should be fixed by setting `shell.profile` to a reasonable value I think, im not sure but i think its that, the semantics of when bash loads what are really confusing

Reply:
If I tried to `source /etc/profile` then it would print variables and so adding path was the best solution I could come up with.

Reply:
Well afaik `/etc/profile` should be empty on NixNG i think, but in the file you would have `export PATH=/bin` or something, its best too look at NixOS does and just copy it, i dont have access to it at the moment though